nginx/tests/integration/test_nginx.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Canonical
+# See LICENSE file for licensing details.
+
+"""Integration tests using a real Juju and charm to test ContainerPath."""
+
+from __future__ import annotations
+
+import jubilant
+import pytest
+
+from conftest import deploy
+
```

Comment:
Since it doesn't look like you're packing a machine charm, I assume that this library (or at least the integration tests) are K8s charm specific. You can skip the machine integration tests being run at all (including packing) by marking all your tests as `k8s_only`.
```suggestion

pytestmark = pytest.mark.k8s_only
```
Or if you want to future proof against forgetting to add `pytestmark` in future integration test files, consider using [pytest_collection_modifyitems](https://docs.pytest.org/en/latest/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems) in your `conftest.py`.

Reply:
yea I don't think this will ever be relevant for machine charms